modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -661,38 +631,40 @@ You can specify more than one tag by separating them using a comma.
       <artifactId>munit-maven-plugin</artifactId>
       <configuration>
       ...
-      <skipMunitTests>True</skipMunitTests>
+      <skipAfterFailure>true</skipAfterFailure>
       ...
     </configuration>
   </plugin>
 </plugins>
 ----
 
+If not specified, this value is `false` by default.
 
-// CONFIGURATION
 
-== To Skip Tests After One Suite Fails
+[[specify-runtime-product]]
+== Specifying The Runtime Product
+
+MUnit allows you to specify the type of runtime in which your applications being tested will run. Accepted values are:
 
-MUnit allows you to skip the rest of the tests if one test suite fails. +
-If not specified, this value is false by default.
+* `MULE` for community edition.
```

Comment:
Community Edition (Is capitalization needed? At least for consistency in Edition)